static/js/partials/recipe-create.html

```diff
@@ -0,0 +1,53 @@
+<div class="jumbo">
+    <h1>New Recipe</h1>
+</div>
+
+<md-content layout-padding="" ng-controller="CreateRecipeCtrl">
+    <div>
+        <form>
+            <div layout-gt-xs="row">
+                <md-input-container class="md-block" flex-gt-xs="">
+                    <label>Author's Full Name</label>
```

Comment:
```suggestion
                    <label>Author's Name</label>
```
removing `Full` again